tools/rosgraph/src/rosgraph/roslogging.py

```diff
@@ -59,7 +59,12 @@ def findCaller(self, dummy=False): # Dummy second arg to match Python3 function
         f = inspect.currentframe()
         if f is not None:
             f = f.f_back
+        else:
+            return file_name, lineno, func_name
```

Comment:
Can you please provide some context why this change was necesary? The same for the `break` a few lines below.

Reply:
without this, roscore won't run on Windows.  I need to investigate more about this change, will revert it from this PR for now.